scripts/ansible/aws/ec2-set-volume-size.yml

```diff
@@ -19,10 +19,9 @@
 
```

Comment:
At some point, this script should be updated to use the same `access_key_id` and `access_key_secret` for consistency

Reply:
agreed - we'll need to pull that through all of the scripts in this folder I think